waiter/doc.go

```diff
@@ -0,0 +1,48 @@
+/*
+Package waiter provides synchronous implementation of asynchronous commands of [client.Client] and [pool.Pool].
+
+Supported operations:
+  - Container put
+  - Container setEacl
+  - Container delete
+  - Object delete
+
+The main component is [Waiter] type. It is using [client.Client] or [pool.Pool] as [Executor] implementation
+for querying async operation and wait some time, to be sure it has effect like container created/deleted etc.
+
+Create waiter instance:
+
+	// each defaultPoolingTimeout waiter checks, the operations made effect.
+	// It happens either we get success or context deadline.
+	defaultPoolingTimeout := 1 * time.Second
+	w := waiter.NewWaiter(pool, defaultPoolingTimeout)
+
+Create container:
+
+	var containerID cid.ID
+	var pp netmap.PlacementPolicy
+	pp.SetContainerBackupFactor(1)
+
+	var rd netmap.ReplicaDescriptor
+	rd.SetNumberOfObjects(1)
+	pp.AddReplicas(rd)
+
+	cont.SetPlacementPolicy(pp)
+	var cmd client.PrmContainerPut
```

Comment:
let's rename it [too](https://github.com/nspcc-dev/neofs-sdk-go/pull/451#discussion_r1233510098)?

Reply:
Done